eng/pipelines/runtime.yml

```diff
@@ -1067,6 +1067,51 @@ extends:
                     eq(variables['coreclrContainsChange'], true),
                     eq(variables['isRollingBuild'], true))
 
+      #
+      # iOS simulator
+      # Build the whole product using CoreCLR and run functional tests
+      #
+      - template: /eng/pipelines/common/platform-matrix.yml
+        parameters:
+          jobTemplate: /eng/pipelines/common/global-build-job.yml
+          helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+          buildConfig: Debug
```

Comment:
Debug build of CoreCLR is very slow. We typically use checked build for testing to save CI resources.